File: src/main/java/org/example/eventhub/dto/order/OrderResponseShort.java
package org.example.eventhub.dto.order;

import java.math.BigDecimal;
import java.util.List;
import org.example.eventhub.dto.ticket.TicketResponseShort;
import org.example.eventhub.enums.OrderStatus;

public record OrderResponseShort(List<TicketResponseShort> tickets, BigDecimal totalPrice, OrderStatus orderStatus) {}

File: src/main/java/org/example/eventhub/dto/order/OrderResponseShort.kt
package org.example.eventhub.dto.order

import org.example.eventhub.dto.ticket.TicketResponseShort
import org.example.eventhub.enums.OrderStatus
import java.math.BigDecimal

data class OrderResponseShort(
    val tickets: List<TicketResponseShort>,
    val totalPrice: BigDecimal,
    val orderStatus: OrderStatus
)
